l3kernel/l3expan.dtx

```diff
@@ -63,32 +63,37 @@
 % \label{sec:l3expan:defining-variants}
 %
 % The definition of variant forms for base functions may be necessary
-% when writing new functions or when applying a kernel function in a
-% situation that we haven't thought of before.
+% when writing new functions or when applying a function in a
+% new situation.
 %
-% Internally preprocessing of arguments is done with functions of the form
-% \cs[no-index]{exp_\ldots{}}.  They all look alike, an example would be
+% Internally preprocessing of arguments is done with functions
+% \cs[no-index]{exp_\meta{\ldots}}.  They all look alike, an example would be
 % \cs{exp_args:NNo}. This function has three arguments, the first and the
 % second are a single tokens, while the third argument should be given
-% in braces. Applying \cs{exp_args:NNo} expands the content of third
-% argument once before any expansion of the first and second arguments.
-% If \cs{seq_gpush:No} was not defined it could be coded in the following way:
-% \begin{verbatim}
-%   \exp_args:NNo \seq_gpush:Nn
-%      \g_file_name_stack
-%      { \l_tmpa_tl }
-% \end{verbatim}
+% in braces. Applying \cs{exp_args:NNo} only expands the content of
+% third argument once, the first and second arguments will eventually
+% be expanded  after.
+% If \cs{seq_gpush:No} was not defined it could be emulated in the following way.
+% \begin{quote}\ttfamily
+%   |\exp_args:NNo \seq_gpush:Nn|\\
+%   |   \g_path_seq|\\
+%   |   { \l_dir_path_tl / \l_base_name_tl }|
```

Comment:
I missed that.
The initial problems are
- using `_seq` is better than `_stack` because the former are far more numerous in the documentation, moreover it refers to `\seq_gpush`
- using `\l_tmpa_tl` is a bit cryptic at the beginning of the documentation
- in that case `\exp_args:NNV` is more appropriate that `\exp_args:NNo` as the documentation says

I can change to
```latex
% |\exp_args:NNo \seq_gpush:Nn|\\
%   |   \g_file_name_seq|\\
%   |   { \l_name_tl .tex }|
```

Reply:
> * using `\l_tmpa_tl` is a bit cryptic at the beginning of the documentation

As there's already `\seq_gpush:Nn`, using a scratch variable seems not too cryptic. I'm OK to both ways.